### PR TITLE
Added support for more options & fixed result preview

### PR DESCRIPTION
**Pull request by asvvvad1**
I found this really helpful since I started compiling Godot from source so I decided to add somethings that I found lacking and fixed the "fixed" result preview. Hope you'll like these changes if not thanks for making it public I'll be using it ^^

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -67,6 +67,15 @@ button:active {
   background-color: var(--button-background-color-active);
 }
 
+aside {
+  max-width: 35rem;
+  display: inline-block;
+}
+
+main {
+	margin-bottom: 1rem;
```
Comment (Calinou) [inline main.css]:
```suggestion
  margin-bottom: 1rem;
```

**Review by Calinou — APPROVED** [review]
Thanks!

**Comment by asvvvad1** [comment]
Thanks for accepting!
Btw, I was thinking about adding option.default so we don't include the options in the result that are set to their defaults

**Comment by Calinou** [comment]
@asvvvad Sure, feel free to open a PR for that :slightly_smiling_face: 

Note that I removed the `debug_symbols` option as its use can be counter-productive. I'd prefer to keep the tool relatively simple so people avoid unintentionally making their binaries larger.

Also, I fixed the preview on the top instead of the bottom of the window as fixing it on the bottom looked strange on my 31.5" monitor.

**Comment by asvvvad1** [comment]
Oh my screen is smaller that's why it looked fine. I tried to zoom out and I get what you mean.
And for debug symbols I didn't know about that I just saw it on by default so I thought disabling would help make it smaller ^^

**Comment by Calinou** [comment]
@asvvvad Debugging symbols are automatically disabled when using `target=release`.

**Pull request merged**